### ISSUE TITLE
privacy concerns need not be tied to privacy laws

### ISSUE THREAD

**Issue by npdoty** [mattgarrish/epub-revision#2267]
The rs and a11y specs note cases where certain behavior might violate privacy laws (and so, implicitly, implementers should take care to protect against that behavior). You might be right about that legal assessment for some jurisdictions, but I don't think that should be the motivation for protecting user privacy or the criterion for which we conclude that user privacy should be protected. We can be concerned for user privacy no matter the jurisdiction and even if violating the user's privacy would be legal in some, many or all jurisdictions.
